packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -228,34 +229,67 @@ export async function handler(event: SQSEvent) {
         log(
           `Calling converter on url ${converterUrl} with params ${JSON.stringify(converterParams)}`
         );
-        const convertPayloadToFHIR = () =>
-          executeWithNetworkRetries(
-            () =>
-              fhirConverter.post(converterUrl, payloadClean, {
-                params: converterParams,
-                headers: { "Content-Type": TXT_MIME_TYPE },
-              }),
-            {
-              // No retries on timeout b/c we want to re-enqueue instead of trying within the same lambda run,
-              // it could lead to timing out the lambda execution.
-              log,
+
+        const partitionedPayload = partitionPayload(payloadClean);
+        const convertPayloadToFHIR = async () => {
```

Comment:
Since we're touching this, can we move these to regular functions, instead of arrow ones, please?

Reply:
Done. But I stayed away from factoring this out into its own function. Hope that's ok

Reply:
Awesome!

Since we prob have another commit coming, can we also make `storePayloadInS3` to be a regular function, please? No need to move to outside of the parent function if you prefer not to do it now.

Reply:
done 👍